dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractCrudController.java

```diff
@@ -1622,7 +1626,7 @@ protected String getEntitySimpleName()
         return entitySimpleName;
     }
 
-    private String calculatePaginationCountKey( User currentUser, List<String> filters, WebOptions options )
+    private String makePaginationCountKey( User currentUser, List<String> filters, WebOptions options )
```

Comment:
_make_ feels misguiding. What about _compute_ or _compose_?

Reply:
I feel that calculate is somewhat exaggerating, it's only concatenating some strings. Compose however is probable even more describing.